TCGA/constants.py

```diff
@@ -6,10 +6,14 @@
 REDUCE_DIMS_RESULTS_FOLDER = Path(__file__).parent / 'reduce_dims_results'
 PLOTS_FOLDER = REDUCE_DIMS_RESULTS_FOLDER / 'plots'
 COLUMN_NAMES_FILE = Path(__file__).parent / 'column_names.json'
+COLUMN_GROUPS_FILE = Path(__file__).parent / 'column_groups.json'
```

Comment:
I just learned about `importlib.resources.files()` recently, which may be a better way to phrase this line. Here it would be something like

```suggestion
COLUMN_GROUPS_FILE = importlib.resources.files(__package__) / 'column_groups.json'
```

which would avoid the awkwardness of referring to the current file's containing directory. In this case you might be able to use a different function from the `importlib.resources` module (maybe `read_text()`), so it might be worth a look.